shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -142,6 +146,17 @@ static bool OnMakeResourceCurrent(FLEViewController* controller) {
   return true;
 }
 
+static bool OnAcquireExternalTexture(FLEViewController* controller,
```

Comment:
Neds a declaration comment. Also, it looks like this should be after the #pragma mark below